src/components/PeoplesPage/PeoplesPage.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonItem } from '../../Person/PersonItem';
+
+export const PeoplesPage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [noPeople, setNoPeople] = useState(false);
```

Comment:
looks like this state is redundant, please remove it and use `!people.length && 'no people'`